styleguide/derek/global-components/ceiling/_ceiling.scss

```diff
@@ -160,7 +160,7 @@ $ceiling-font-size: $navigation-font-size-medium;
 .ceiling-dropdownToggle {
   color: $white;
   display: block;
-  font-size: $navigation-font-size-medium;
+  font-size: 1.15rem;
```

Comment:
This looks like it's only changing the signup button, not the login button.  

We probably need to make this change to `.ceiling-menuInline-link` as well.